packages/federatorai/federatorai.patch

```diff
@@ -0,0 +1,18 @@
+diff -x '*.tgz' -x '*.lock' -uNr packages/federatorai/charts-original/Chart.yaml packages/federatorai/charts/Chart.yaml
+--- packages/federatorai/charts-original/Chart.yaml
++++ packages/federatorai/charts/Chart.yaml
+@@ -1,5 +1,8 @@
+ apiVersion: v1
+ appVersion: 4.5.1-ga
++annotations:
++  catalog.cattle.io/certified: partner
++  catalog.cattle.io/release-name: federatorai
```

Comment:
Please add the following annotation for the name you want displayed in the UI.
`catalog.cattle.io/display-name: Federator AI`
I used Federator AI as an example, but you can set it to any name you think is best.

Reply:
Fixed by forcibly commit.